### PR TITLE
feat: add templates

### PR DESCRIPTION
**Pull request by shamsartem**
fix: fluence run --dry

### CODE REVIEW

File: src/lib/paths.ts [hunk 1]
```diff
@@ -120,17 +122,51 @@ export const ensureFluenceAquaServicesDir = async (): Promise<string> =>
 export const ensureFluenceAquaDeployedAppPath = async (): Promise<string> =>
   path.join(await ensureFluenceAquaDir(), DEPLOYED_APP_AQUA_FILE_NAME);
 
-export const ensureFluenceJSAppPath = (fluenceJSDir: string): string =>
-  path.join(fluenceJSDir, APP_JS_FILE_NAME);
+// JS
 
-export const ensureFluenceJSDeployedAppPath = (fluenceJSDir: string): string =>
-  path.join(fluenceJSDir, DEPLOYED_APP_JS_FILE_NAME);
+export const ensureFluenceJSAppPath = async (
```
Comment (coder11) [inline src/lib/paths.ts:127]: Are these paths configurable? I mean JS and TS destination paths.

File: src/lib/const.ts [hunk 2]
```diff
@@ -142,52 +160,52 @@ import "@fluencelabs/aqua-lib/builtin.aqua"
 
 -- import App from "deployed.app.aqua"
 -- import Adder from "services/adder.aqua"
--- export App, add_one
+-- export App, addOne
```
Comment (coder11) [inline src/lib/const.ts:163]: If you are doing snake_case to camelCase renaming only for the sake of making generated TS files look nice than I think we are doing it in the wrong place. This should be done inside aqua compiler.

File: src/lib/paths.ts (hunk 1 above)
Comment (coder11) [reply src/lib/paths.ts:127]: @DieMyst WDYT?

@folex  WDYT about recommended naming notation?

File: src/lib/const.ts (hunk 2 above)
Comment (coder11) [reply src/lib/const.ts:163]: @DieMyst WDYT?

@folex WDYT about recommended naming notation?

Comment (folex) [reply src/lib/const.ts:163]: I like snake case. But I don't care much.

File: src/lib/paths.ts (hunk 1 above)
Comment (shamsartem) [reply src/lib/paths.ts:127]: now yes

File: src/lib/const.ts (hunk 2 above)
Comment (shamsartem) [reply src/lib/const.ts:163]: I think compiler shouldn't do that. It's just camelCase is a lot more popular among js developers so I think people who compile to js would prefer using camelCase themselves

File: src/lib/paths.ts (hunk 1 above)
Comment (coder11) [reply src/lib/paths.ts:127]: awesome

File: src/lib/const.ts (hunk 2 above)
Comment (coder11) [reply src/lib/const.ts:163]: Our built-ins library is using snake case in function naming. So if a developer would be using camelCase they would inevitably face style inconsistencies

**Review by coder11 — APPROVED** [review]
Feel free to merge the changes if you need them ASAP. But I would suggest to come to some sort of an agreement on naming style and fix the aqua template accordingly.

**Comment by shamsartem** [comment]
I will merge this for now. If we decide something different - I will just do another PR with a fix

**Pull request merged**